assets/config/es/footer.js

```diff
@@ -45,7 +45,7 @@ const esFooter = {
       'https://www.freecodecamp.org/espanol/news/product-design-explicado-en-un-espanol-sencillo/',
     article14title: 'Digital Design',
```

Comment:
```suggestion
    article14title: 'Diseño Digital',
```

One more - should this title be Spanish?

Reply:
🤦🏽‍♂️ Thanks Nick